admin/view/template/module/tawkto.twig

```diff
@@ -174,5 +337,72 @@
 
 		});
 	}
+
+	jQuery(document).ready(function () {
+		if (jQuery("#always_display").prop("checked")) {
+			jQuery('.show_specific').prop('disabled', true);
+		} else {
+			jQuery('.hide_specific').prop('disabled', true);
+		}
+
+		jQuery("#always_display").change(function () {
+			if (this.checked) {
+				jQuery('.hide_specific').prop('disabled', false);
+				jQuery('.show_specific').prop('disabled', true);
+			} else {
+				jQuery('.hide_specific').prop('disabled', true);
+				jQuery('.show_specific').prop('disabled', false);
+			}
+		});
+
+		// process the form
+		jQuery('#module_form').submit(function (event) {
+			var store_id = jQuery('select[name="store_id"]').val();
+			var form = jQuery(this);
+
+			$path = '{{url.set_options_url}}';
+			jQuery.post($path, {
+				store: store_id,
+				options: form.serialize(),
+			}, function (r) {
+				if (r.success) {
+					$('#optionsSuccessMessage').toggle().delay(3000).fadeOut();
+
+					// Update saved options
+					var fields = form.serializeArray();
+					for (store of storeHierarchy) {
+						if (store.id !== store_id) {
+							continue;
+						}
+
+						store.display_opts = {
+							'always_display': false,
+							'show_onfrontpage': false,
+							'show_oncategory': false,
+							'show_oncustom': [],
+							'hide_oncustom': [],
+						};
+
+						for (field of fields) {
+							if (field.name === 'show_oncustom') {
+								store.display_opts['show_oncustom'] = field.value.replaceAll('\r', '\n').split('\n').filter(Boolean);
+								continue;
+							}
+
+							if (field.name === 'hide_oncustom') {
+								store.display_opts['hide_oncustom'] = field.value.replaceAll('\r', '\n').split('\n').filter(Boolean);
+								continue;
+							}
+
+							store.display_opts[field.name] = field.value === '1';
```

Comment:
This value when render as json in storeHierarchy is boolean but updated as string '1'

Reply:
serializeArray() only returns checked inputs, hence will change this to assign to true